qiskit/pulse/commands/instruction.py

```diff
@@ -220,12 +222,17 @@ def draw(self, dt: float = 1, style: Optional['SchedStyle'] = None,
 
         from qiskit import visualization
 
+        if channels_to_plot:
+            warnings.warn('The parameter "channels_to_plot" is being replaced by "channels"'
```

Comment:
We should be able to do this instead:
```suggestion
            warnings.warn('The parameter "channels_to_plot" is being replaced by "channels"',
```
and then on the next line, just `DeprecationWarning, 3)`